recognition/include/pcl/recognition/face_detection/face_common.h

```diff
@@ -67,7 +69,7 @@ namespace pcl
     };
 
     template<class FeatureType>
-    class RFTreeNode
+    class EIGEN_ALIGN16 RFTreeNode
```

Comment:
Why do we need this?

Reply:
Currently, I am not sure if this macro is really required. It is possible that on some system using some compilers it is required to have the stack aligned.
According to Eigen this issue has only been met on Windows with the GCC compiler (bug). The bug might been fixed in GCC 4.5, see:

https://eigen.tuxfamily.org/dox/group__TopicWrongStackAlignment.html